<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>23</width>
        <height>18</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="playBar" name="control" native="true">
      <property name="minimumSize">
       <size>
        <width>741</width>
        <height>41</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="musicStatus" name="status" native="true">
      <property name="minimumSize">
       <size>
        <width>741</width>
        <height>51</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>31</width>
        <height>18</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QScrollArea" name="scrollArea">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>921</width>
         <height>398</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
   <zorder>control</zorder>
   <zorder>status</zorder>
   <zorder>scrollArea</zorder>
   <zorder>horizontalSpacer</zorder>
   <zorder>horizontalSpacer_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>941</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMyTunes">
    <property name="title">
     <string>myTunes</string>
    </property>
    <addaction name="actionAdd_music"/>
    <addaction name="actionChange_music_directory"/>
   </widget>
   <addaction name="menuMyTunes"/>
  </widget>
  <action name="actionMyMusic">
   <property name="text">
    <string>myMusic</string>
   </property>
  </action>
  <action name="actionAdd_music">
   <property name="text">
    <string>add music</string>
   </property>
  </action>
  <action name="actionChange_music_directory">
   <property name="text">
    <string>change music directory</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>playBar</class>
   <extends>QWidget</extends>
   <header>playbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>musicStatus</class>
   <extends>QWidget</extends>
   <header>musicstatus.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
